lib/omise/charge.ex

```diff
@@ -254,12 +257,16 @@ defmodule Omise.Charge do
   ## Examples
 
       Omise.Charge.capture("chrg_test_4xso2s8ivdej29pqnhz")
+      Omise.Charge.capture("chrg_test_4xso2s8ivdej29pqnhz",key: "skey_test_123")
+      Omise.Charge.capture("chrg_test_4xso2s8ivdej29pqnhz",[capture_amount: 3000])
+      Omise.Charge.capture("chrg_test_4xso2s8ivdej29pqnhz",[key: "skey_test_123"],[capture_amount: 3000])
```

Comment:
This example needs to be updated to:

```
Omise.Charge.capture("chrg_test_4xso2s8ivdej29pqnhz", key: "skey_test_123", capture_amount: 3000)
```

The opts can be written with or without brackets.

Reply:
Yeah will also update